metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
There's a Python [module](https://github.com/apache/incubator-metron/tree/master/metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/KIBANA/4.5.1/package/scripts/dashboard) that I wrote as part of the MPack that does the work there. I have an outstanding item to write up it's basic usage, but can't find a good place in the code. Ideas?